README.md

```diff
@@ -148,14 +201,14 @@ Write ruby code that will create a person.
 
 Your answer:
 ```
-Replace this with your answer
+bob = Person.new("bob", 45)
 ```
```

Comment:
this would only work for a plain old ruby object. for creating any object from a class that inherits from ActiveRecord, you have to use key value pairs i.e. `bob = Person.new(name: "bob", age: 45)`